src/main/java/seedu/address/commons/core/GuiSettings.java

```diff
@@ -8,7 +8,8 @@
  * A Serializable class that contains the GUI settings.
  */
 public class GuiSettings implements Serializable {
-
+    // TODO change the default height and width to something appropriate
```

Comment:
What should our default size be?
maybe whole screen since the calendar is so big